dev-libs/wlc/metadata.xml

```diff
@@ -9,8 +9,4 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-	<use>
-		<flag name="systemd">Enable support for systemd-logind.</flag>
-		<flag name="X">Enable X11 backend and XWayland support.</flag>
```

Comment:
I was just thinking that the descriptions are unnecessary, because these are standard global use flags.

Yeah, it is an unrelated change. I should probably leave them for now, because it is irrelevant to this pull request.